plugins/zsh-autosuggestions/README.MD

```diff
@@ -0,0 +1 @@
+As you type commands, you will see a completion offered after the cursor
```

Comment:
I think we can make a better README than this. Some users won't know that this has been "adopted" from outside. For them, this is all they see. 

Look at the original repo: https://github.com/zsh-users/zsh-autosuggestions. The README contains so many things that I don't think just a line can cover them.